api/application.proto

```diff
@@ -55,6 +55,10 @@ message Applications {
   repeated Application applications = 1;
 }
 
+message DevEUI {
```

Comment:
The naming is problematic here, since we would be inconsistent with other usages. I think `RequestDevEUIResponse` would clear that up.